.github/workflows/conformance-datapath.yaml

```
@@ -227,7 +227,7 @@ jobs:
             --nodes-without-cilium=kind-worker3 \
             --helm-set-string=kubeProxyReplacement=${{ matrix.kpr }} \
             --helm-set=bpf.masquerade=false"
-          TUNNEL="--helm-set-string=tunnel=vxlan"
+          TUNNEL="--helm-set-string=tunnel=${{ matrix.tunnel }}"
           if [ "${{ matrix.tunnel }}" == "disabled" ]; then
             TUNNEL="--helm-set-string=tunnel=disabled --helm-set-string=autoDirectNodeRoutes=true --helm-set-string=ipv4NativeRoutingCIDR=10.244.0.0/16"
```

for direct routing this gets set twice:

```suggestion
            TUNNEL="--helm-set-string=autoDirectNodeRoutes=true --helm-set-string=ipv4NativeRoutingCIDR=10.244.0.0/16"
```

---

Note the `TUNNEL` variable is overwritten. We're **not** using `TUNNEL+=`. So AFAICS, it should only be set once in native routing mode.